apps/backend/src/scripts/parsetdi34.ts

```diff
@@ -0,0 +1,16 @@
+import { handler } from '../lambdas/parseFlatFile';
+
+const formatOutput = (filepath: any) => {
```

Comment:
(this is a temporary file since we won't be running scripts manually in prod - for dev only, as with the parsetdi17)